Automattic/wp-calypso
# Stats: Adding Devices module

Related to #89296

## Proposed Changes

* adding a new component for the Devices module
* because the API is not yet finalised, I'm using the UTM endpoint as a placeholder

## Testing Instructions



* open the live branch
* verify that the Devices module is not visible at the bottom of the Traffic page
* apply `stats/devices` feature flag e.g. `/stats/day/<site_id>?flags=stats/devices`
* verify that the new module is visible

## Pre-merge Checklist
